nipype/interfaces/ants/registration.py

```diff
@@ -502,16 +503,32 @@ def _formatMetric(self, index):
             # dict-comprehension only works with python 2.7 and up
             #specs = [{k: v[i] for k, v in items} for i in indexes]
             specs = [dict([(k, v[i]) for k, v in items]) for i in indexes]
+            specs = list()
```

Comment:
Did you rebase/merge master properly? This code was already merged from a recent @hjmjohnson PR https://github.com/nipy/nipype/blob/master/nipype/interfaces/ants/registration.py#L520